modules/@apostrophecms/modal/ui/apos/components/AposDocsManagerToolbar.vue

```diff
@@ -135,7 +135,7 @@ export default {
     },
     checkedTypes: {
       type: Array,
-      default: () => []
+      default: null
```

Comment:
We want `checkedTypes` to exist only if it has been passed. otherwise it's null.